core/jreleaser-model-impl/src/main/java/org/jreleaser/model/internal/validation/packagers/ChocolateyPackagerValidator.java

```diff
@@ -250,6 +250,10 @@ public static void postValidateChocolatey(JReleaserContext context, Distribution
 
         Project project = context.getModel().getProject();
 
+        if (distribution.getTags().isEmpty()) {
+            errors.configuration(RB.$("validation_must_not_be_empty", "tags"));
```

Comment:
Should probably be more specific with the missing property name. Please update it to "distribution.tags". Thanks.

Reply:
Of course, I was a tad on the fence between the two myself.